drivers/hmis/app/models/hmis/hud/youth_education_status.rb

```diff
@@ -11,6 +13,7 @@ class Hmis::Hud::YouthEducationStatus < Hmis::Hud::Base
   include ::Hmis::Hud::Concerns::Shared
   include ::Hmis::Hud::Concerns::EnrollmentRelated
   include ::Hmis::Hud::Concerns::ClientProjectEnrollmentRelated
+  include ::Hmis::MarkClientAsDirtyBehavior
```

Comment:
I don't think this is necessary, this record is collected on a hud assessment (which gets stored as a custom assessment) so the CustomAssessment model trigger should be sufficient. We also don't have any use-case yet for  using this model for eligibility